app/src/app/components/routes/ProtectedRoute.tsx

```diff
@@ -13,6 +14,17 @@ export const ProtectedRoute = () => {
   const { pathname } = useLocation();
   const expiryMessage = "You have been signed out because your session expired. Please log in.";
 
+  const navigateToLogin = (qs = "") => {
+    // navigate to logged out screen, either using react routing or external navigation, depending on whether we're
+    // using pre-auth, which may require logout action from auth provider
+    if (authConfig?.enablePreAuthLogin) {
+      const href = `${process.env.PUBLIC_URL}/logout`;
```

Comment:
Is `${process.env.PUBLIC_URL}/logout` an endpoint in montagu? Or where is the logic that logs out the montagu session when packit logs out?

Reply:
I was expecting to see something here relating to 

> in this case we pass a "loggingOut" flag so that Montagu knows that it has to log out too

but I don't see how/where we pass this to Montagu.

Reply:
Ah yes, sorry, I put that into the nginx config in the proxy and didn't update the description here. I thought it would be better to have a generic `logout` route from here, and then let the preauthenticating application deal with it appropriately - here by redirecting with "loggingOut=1":
line 155 in nginx.montagu.conf:
https://github.com/vimc/montagu-proxy/pull/97/files#diff-76948d98bd1b1f9548f44ff60e6d989cc95a8aeafe5f52555b2aa7dbf4636059R155

Reply:
Would be good to link there from a comment on this code, so others can follow the breadcrumbs in the future.